copc/index.qmd

```diff
@@ -0,0 +1,12 @@
+---
+title: Cloud-Optimized Point Clouds (COPC)
+subtitle: An Introduction to Cloud-Optimized Point Clouds (COPC)
+---
+
+The LAS(er) file format is designed to store 3-dimension point cloud data typically collected from [LiDAR](https://en.wikipedia.org/wiki/Lidar).
```

Comment:
Will most people understand the `(er)` after LAS? I see Wikipedia defines `LAS (LASer)`, but without looking that up, this `LAS(er)` made me confused.